config/sample_hanatpcc_config.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0"?>
+<parameters>
+	
+    <!-- Connection details -->
+    <dbtype>saphana</dbtype>
+    <driver>com.sap.db.jdbc.Driver</driver>
+    <DBUrl>jdbc:sap://ottertune-dev.pdl.cmu.edu:39015/tpcc</DBUrl>

```

Comment:
Don't use public domains. See other examples.

Reply:
changed to server